src/components/TopMenu.js

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react'
 import { Link, graphql } from 'gatsby'
-import { RichText } from 'prismic-reactjs'
+import { PrismicLink, PrismicText } from '@prismicio/react'
```

Comment:
If we're gonna use `PrismicLink` only, we can get rid of gatsby's `Link` right?
Why is it best to use ours?

Reply:
Correct; we can remove Gatsby's Link. I'll update it to use `<PrismicLink>` instead.

It is best to use `<PrismicLink>` because it will automatically handle internal (e.g. `/about`) and external (e.g. `https://prismic.io`) links. This is especially important when links come from a CMS since an editor could provide either an internal or external link without the developer knowing. Gatsby's `<Link>` component only handles internal links.